code/modules/vending/liberation.dm

```diff
@@ -28,6 +28,6 @@
 					/obj/item/reagent_containers/food/snacks/burger/superbite = 3) //U S A
 	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
 	resistance_flags = FIRE_PROOF
-	default_price = 150
-	extra_price = 500
+	default_price = 0 //liberty and justice for all
+	extra_price = 0
```

Comment:
The few maploaded liberation vendors should already be free as they aren't initialized on the station z levels... but if you are in dire needs of a freeloader version of this rare vendor, please make a subtype, or VV the thing in game.

Reply:
AMERICAN JUSTICE HAS NO PRICE OTHER THAN THE BLOOD AND SWEAT OF OUR COMPATRIOTS